R/install.R

```diff
@@ -129,11 +129,14 @@ install_deps <- function(pkg = ".",
                          build = TRUE,
                          build_opts = c("--no-resave-data", "--no-manual", " --no-build-vignettes"),
                          ...) {
+  remotes::update_packages("roxygen2")
```

Comment:
Why did you add this?

Reply:
I wanted to align the internals of `install_dev_deps()` and `install_deps()`. Happy to remove the changes to `install_deps()` if it shouldn't be done as part of this PR.

Reply:
It is in `install_dev_deps()` because you need roxygen2 when developing packages, but you definitely don't want to always install it whenever you are installing any dependency.